lib/main.js

```diff
@@ -102,11 +109,25 @@ export default {
         const { child, worker, subscription } = spawnWorker()
         this.worker = worker
         this.subscriptions.add(subscription)
+
         child.on('exit-linter', shouldLive => {
           this.worker = null
           // Respawn if it crashed. See atom/electron#3446
           if (shouldLive) {
-            initializeWorker()
```

Comment:
Crashes don't happen all the time for every user, IMO instead of having a config and all of that, we could just add a default delay of 5 seconds here and it'll work good